kr_mesh_visualization/launch/test_launch.py

```diff
@@ -0,0 +1,39 @@
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration, TextSubstitution
+from launch_ros.actions import Node
+
+def generate_launch_description():
+
+  input_topic_launch_arg = DeclareLaunchArgument(
+    "~/input",
+    default_value=TextSubstitution(text="/quadrotor/odom")
+  )
+  
+  new_frame_id_launch_arg = DeclareLaunchArgument(
+    "new_frame_id", default_value=TextSubstitution(text="")
+  )
+
+  mesh_visualization_node = Node(
+    package="kr_mesh_visualization",
+    executable="kr_mesh_visualization",
+    name="mesh_visualization",
+    output="screen",
+    remappings=[
+      ("~/input", LaunchConfiguration("~/input"))
+    ],
+    parameters=[
+      {"mesh_resource": "package://kr_mesh_visualization/mesh/hummingbird.mesh"},
+      {"color/r": 0.0},
+      {"color/g": 0.0},
+      {"color/b": 1.0},
+      {"color/a": 0.7},
+      {"new_frame_id": LaunchConfiguration("new_frame_id")}
+    ]
+  )
+
+  return LaunchDescription([
+    input_topic_launch_arg,
+    new_frame_id_launch_arg,
+    mesh_visualization_node
+  ])
```

Comment:
Newline https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
I'll ensure that there's a newline in each file